src/main/java/org/testeditor/fixture/host/util/LineReader.java

```diff
@@ -0,0 +1,92 @@
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+package org.testeditor.fixture.host.util;
+
+import org.testeditor.fixture.host.locator.Locator;
+import org.testeditor.fixture.host.locator.LocatorByStartStop;
+import org.testeditor.fixture.host.locator.LocatorByWidth;
+
+import java.util.List;
+
+/**
+ * This is a utility class for reading multiple lines on a Mainframe Screen. For
+ * reading multiple lines, we need a start point and an end point, these start
+ * and end points can received through a start row and start column and end row
+ * and end column.
+ *
+ */
+public class LineReader {
+
+    /**
+     * 
+     * @param lines
+     *            the rows which should be read.
+     * @param locator
+     *            see {@link Locator} the start and endpoint representation of
+     *            String to be read.
+     * @return the excerpted String from start to endpoint
+     */
+    public String readMultilines(List<String> lines, Locator locatorIn) {
+        StringBuffer sb = new StringBuffer();
+
+        LocatorByStartStop locator = (LocatorByStartStop) locatorIn;
+
+        int startRow = locator.getStartRow();
+        int endRow = locator.getEndRow();
+
+        for (int i = startRow - 1; i < endRow; i++) {
+            String line = lines.get(i).replace("data:  ", "");
+            if (i == startRow - 1) {
+                sb.append(line.substring(locator.getStartColumn() - 1));
+            } else if (i == endRow - 1) {
+                sb.append(line.substring(0, locator.getEndColumn()));
+            } else {
+                sb.append(line);
+            }
+        }
+        return sb.toString();
+    }
+
+    /**
+     * @param line
+     *            the row which should be read.
+     * @param locator
+     *            see {@link Locator} the start and endpoint representation of
+     *            String to be read.
+     * @return the excerpted String from start to endpoint.
+     */
+    public String readSingleLine(String line, Locator locatorIn) {
+        String lineReplaced = line.replace("data:  ", "");
+        LocatorByStartStop locator = (LocatorByStartStop) locatorIn;
+        return (lineReplaced.substring(locator.getStartColumn() - 1, locator.getEndColumn()));
+    }
+
+    /**
+     * @param line
+     *            the row which should be read.
+     * @param locator
+     *            see {@link Locator} the start and endpoint representation of
+     *            String to be read.
+     * @return the excerpted String from start to endpoint.
+     */
+    public String readSingleLineWidth(String line, Locator locatorIn) {
+        int offset = -1;
+        String lineReplaced = line.replace("data:  ", "");
```

Comment:
could be put into separate function, since this replacement is used all over the place